Bottom: 44f8f1d4ce3fc0356402ae05bbe8b63c973a4865
Top:    27bfde2fb2ab4ee3033c6b465d55530a614e6bed
Author: Sean Hefty <sean.hefty@intel.com>
Date:   2012-06-26 16:40:05 -0700

Refresh of v6only

---

diff --git a/src/rsocket.c b/src/rsocket.c
index 860c63e..394fed4 100644
--- a/src/rsocket.c
+++ b/src/rsocket.c
@@ -1933,6 +1933,7 @@ int rgetsockopt(int socket, int level, int optname,
 			ret = ENOTSUP;
 			break;
 		}
+		break;
 	case SOL_RDMA:
 		switch (optname) {
 		case RDMA_SQSIZE:
